talk/owt/sdk/base/customizedaudiodevicemodule.cc

```diff
@@ -577,9 +577,15 @@ int CustomizedAudioDeviceModule::GetRecordAudioParameters(
   return _ptrAudioDevice->GetRecordAudioParameters(params);
 }
 #endif  // WEBRTC_IOS
-void CustomizedAudioDeviceModule::CreateOutputAdm(){
+
+void CustomizedAudioDeviceModule::CreateOutputAdm() {
   if (_outputAdm == nullptr) {
+#if defined(WEBRTC_INCLUDE_INTERNAL_AUDIO_DEVICE)
```

Comment:
shouldn't it be #if !defined(WEBRTC_INCLUDE_INTERNAL_AUDIO_DEVICE) ?

Reply:
Fixed.